integration-tests/pom.xml

```diff
@@ -43,6 +43,16 @@
         <hadoop.s3.impl>org.apache.hadoop.fs.s3a.S3AFileSystem</hadoop.s3.impl>
     </properties>
 
+    <dependencyManagement>
```

Comment:
can we add a note as to which dependencies still rely on snakeyaml 1.33?

Reply:
done